airqo-platform/AirQo-frontend
# [NetManager] Mobile Responsivenes part 2

#### Summary of Changes (What does this PR do?)
- Mobile responsiveness for device view page
- Fixes these issues
   -- [issue 222](https://github.com/airqo-platform/AirQo-frontend/issues/222)
  -- [issue 212](https://github.com/airqo-platform/AirQo-frontend/issues/212)
#### Is this change ready to hit production in its current state?

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?
* Fetch this branch locally
* Install requirements using `npm install`
* Start Application using `npm run stage-mac` (MacOs) or `npm run stage-pc` (Windows PC)
* Navigate the device details overview, the page should be responsive.
* The issues stated above should be resolved.

#### What are the relevant tickets?
- [PLAT-483](https://airqoteam.atlassian.net/browse/PLAT-483)

#### Screenshots (optional)
